tests/legacy/unit-tests/templates/functions.php

```diff
@@ -178,4 +178,16 @@ function( $gateways ) {
 
 		$this->assertEquals( $expected_html, $actual_html );
 	}
+
+	public function test_hidden_field() {
+		$actual_html = woocommerce_form_field('test',
+		array(
+			'type' => 'hidden',
+			'id' => 'test_field',
```

Comment:
It would be a little more complete to also verify that the input classes and custom attributes all work too.